docs/source/install/profile.rst

```diff
@@ -103,6 +134,12 @@ For this profile to work, you need to download the :ref:`PIConGPU source code <i
 Taurus (TU Dresden)
 -------------------
 
+**System overview:** `link <https://tu-dresden.de/zih/hochleistungsrechnen/hpc>`_
+
+**User guide:** `link <https://doc.zih.tu-dresden.de/hpc-wiki/bin/view/Compendium/SystemTaurus>`_
+
+**Production directory:** ``/scratch/$USER/`` and ``/scratch/$proj/``
```

Comment:
`$proj` is not defined by default

Reply:
you could write:
```bash
/scratch/`groups | awk '{print $1}'`/
```

Reply:
nah, we define `$proj` in the Titan profiles already and should add it to the taurus `picongpu.profile` as well.

Reply:
I am fine with that.

Reply:
ok, will do as a follow-up